pkg/sql/executor.go

```diff
@@ -28,6 +28,7 @@ import (
 
 	pb "sqlflow.org/sqlflow/pkg/server/proto"
 	"sqlflow.org/sqlflow/pkg/sql/codegen/tensorflow"
+	"sqlflow.org/sqlflow/pkg/sql/codegen/xgboost"
```

Comment:
```
"sqlflow.org/sqlflow/pkg/sql/codegen/xgboost"
xgb "sqlflow.org/sqlflow/pkg/sql/codegen/xgboost"
```
Looks like the package has been imported twice. Not sure if there is any side effect.